Options.py

```diff
@@ -955,6 +955,14 @@ def as_dict(self, *option_names: str, casing: str = "snake") -> typing.Dict[str,
                 raise ValueError(f"{option_name} not found in {tuple(type(self).type_hints)}")
         return option_results
 
+    def get_value(self, option_name: str) -> typing.Any:
```

Comment:
I'm also not feeling good about this. It could lead people to do things in ways that aren't type safe.

If this is needed, I would like to see in the doc string a recommendation against using it.

Reply:
What would you propose as a way to access a single option by name?

Reply:
In the actual context of when this is used, I think it's fine. But yeah, this is just ripe for people copy pasting it, so I can agree with a doc string recommending against that, at minimum.

Reply:
> What would you propose as a way to access a single option by name?

PR incoming

Reply:
> What would you propose as a way to access a single option by name?

https://github.com/alwaysintreble/Archipelago/pull/21